hommod_rest/tasks.py

```diff
@@ -18,7 +20,8 @@ def remodel_oldest_hg():
 
     sequence = fasta.values()[0]
 
-    touch_file(fasta_path)
+    # Touch the fasta first:
```

Comment:
Almost. This comment says what you are doing (touching the file), not why you are doing it (to update the file's modified time to the current time.